patches/kaggle_module_resolver.py

```diff
@@ -1,12 +1,13 @@
 import os
 import re
+import kagglehub
 
 from tensorflow_hub import resolver
 
-url_pattern = re.compile(r"https?://([a-z]+\.)?kaggle.com/models/[^\\/]+/(?P<model>[^\\/]+)/frameworks/(?P<framework>[^\\/]+)/variations/(?P<variation>[^\\/]+)/versions/(?P<version>[0-9]+)$")
+url_pattern = re.compile(r"https?://([a-z]+\.)?kaggle.com/models/(?P<owner>[^\\/]+)/(?P<model>[^\\/]+)/frameworks/(?P<framework>[^\\/]+)/variations/(?P<variation>[^\\/]+)/versions/(?P<version>[0-9]+)$")
 
 def _is_on_kaggle_notebook():
-    return os.getenv("KAGGLE_CONTAINER_NAME") != None
+    return os.getenv("KAGGLE_KERNEL_RUN_TYPE") != None
```

Comment:
Using the same environment variable than `kagglehub`. Both are set so this is a NoOp: https://github.com/Kaggle/kagglehub/blob/c1b6e00226ca952b1d93d05089980dfe233f0163/src/kagglehub/kaggle_cache_resolver.py#L29C1-L29C1